src/isString.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+const isString = (val) => typeof val === 'string'
```

Comment:
would fail of strings constructed with `new String(...)`. Is that something we should care about?

Reply:
Good catch, lets do:

```js
const isString = (val) => typeof val === 'string' || val instanceOf String
```